toxics/slicer.go

```diff
@@ -37,10 +37,13 @@ func (t *SlicerToxic) chunk(start int, end int) []int {
 		return []int{start, end}
 	}
 
+	mid := start + (end-start)/2
 	// +1 in the size variation to offset favoring of smaller
 	// numbers by integer division
 	//#nosec
```

Comment:
```suggestion
	//#nosec
	mid := start + (end-start)/2 + rand.Intn(2)

```

Reply:
I propose keeping the comment. It is accurate to the code (creating a random on {0, 1}) and is helpful because otherwise I could not have guessed what the random integer was for.

Reply:
The comment suggested to always +1 and only related to part `(end-start)/2`. Currently the part of calculation size is bigger and comment did not cover all other parts. 

That's the reason we decided to remove it.

Other solution is to remove `rand.Intn(2)`

```suggestion
	mid := start + (end-start)/2
```